CRM/Core/Smarty.php

```diff
@@ -340,4 +342,18 @@ private function getLocale() {
     return 'en_US';
   }
 
+  /**
+   * Get the compile_check value.
+   *
+   * @return bool
+   */
+  private function isSmartyCompiled() {
```

Comment:
Perhaps the name should be isCheckSmartyIsCompiled() because I think it's misleading